modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vmc/installing-restricted-network-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+ifeval::["{context}" == "installing-vmc"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-vmc-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-vmc-network-customizations"]
+:ipi:
+endif::[]
+
+[id="setting-up-vmc-for-vsphere_{context}"]
+= Setting up VMC for vSphere
+
+You can install {product-title} on VMware Cloud (VMC) on AWS hosted vSphere clusters to enable applications to be deployed and managed both on-premises and off-premises, across the hybrid cloud.
```

Comment:
Even though I disagree with it, I'm pretty sure we use "on-premise" and "off-premise" singular at Red Hat.

https://redhat-documentation.github.io/supplementary-style-guide/#on-premise